packages/core/hardhat.config.js

```diff
@@ -14,4 +14,11 @@ const configOverride = {
   }
 };
 
-module.exports = getHardhatConfig(configOverride);
+// `getHardhatConfig` has a `configOverrides` parameter, but we can't use it
+// since it does a shallow merge and we want to override a single nested field.
+const config = getHardhatConfig(configOverride);
+
+// The default is 199. FiXME - comment out this line to see what goes wrong:
+config.solidity.settings.optimizer.runs = 25;
```

Comment:
Nit: these flags should be set in the global config defined in the common package rather than overridden here.